base/common/python/pki/client.py

```diff
@@ -424,12 +424,14 @@ def connect(self):
         logger.info('Connecting to %s', urllib.parse.urlunparse(self.url))
 
         self.info = self.info_client.get_info()
+        self.api_path = self.info_client.get_api_path()
+        self.api_version = self.info_client.get_api_version()
 
         self.server_version = pki.util.Version(self.info.version)
         logger.debug('- server version: %s', self.server_version)
 
         # if not specified, set REST API version and path based on server version
-        if not self.api_version:
+        if not self.api_version and not self.api_path:
```

Comment:
Version 11.5.0 has some v2 APIs [implemented](https://github.com/dogtagpki/pki/tree/v11.5/base/server/src/main/java/org/dogtagpki/server)  but it is not complete and should not be supported. 
So even the api_path v2 is available it cannot be used if version is not >= 11.5.0. This should be considered in this check